### PR TITLE
Add loadHTMLString() method to WebViewProxy.

### PR DESCRIPTION
**Pull request by Kyome22**
We want to use `loadHTMLString(_:baseURL:)` method.

### CODE REVIEW

File: Sources/WebUI/WebViewProxy.swift [hunk 1]
```diff
@@ -92,6 +92,14 @@ public final class WebViewProxy: ObservableObject {
         webView?.wrappedValue.load(request)
     }
 
+    /// Loads the contents of the specified HTML string and navigates to it.
+    /// - Parameters:
+    ///   - string: The string to use as the contents of the webpage.
+    ///   - baseURL: The base URL to use when the system resolves relative URLs within the HTML string.
+    public func loadHTMLString(_ string: String, baseURL: URL?) {
+        webView?.wrappedValue.loadHTMLString(string, baseURL: baseURL)
+    }
```
Comment (ichiho-ojima) [inline Sources/WebUI/WebViewProxy.swift:101]: Is it difficult to add UI tests for this method?

Comment (b1ackturtle) [inline Sources/WebUI/WebViewProxy.swift:101]: Is it difficult to add UI tests for this method?

Comment (Kyome22) [reply Sources/WebUI/WebViewProxy.swift:101]: @b1ackturtle I did it.
[0ba2d5a](https://github.com/cybozu/WebUI/pull/25/commits/0ba2d5a8b25ac689ed24f0d5e901a1f200520fe0)

Comment (Kyome22) [reply Sources/WebUI/WebViewProxy.swift:101]: https://github.com/cybozu/WebUI/actions/runs/11365353567
Passed the test.

**Review by b1ackturtle — APPROVED** [review]

**Pull request merged**